pilot/pkg/proxy/resolve.go

```diff
@@ -18,30 +18,42 @@ import (
 	"context"
 	"fmt"
 	"net"
-	"strings"
 	"time"
 
 	"istio.io/istio/pkg/log"
 )
 
-// ResolveAddr resolves an authority address to an IP address
+// ResolveAddr resolves an authority address to an IP address. Incoming
+// addr can be an IP or name. If IPv6 address, the IP part must be enclosed
```

Comment:
IP or name => IP address or hostname
If IPV6, the => If addr is an IPv6 address, the

Reply:
Will update.